FutureNorthants/northants-website
# Frontend support for GoogleMap slice

## Setup 

Relies upon these two PRs:
- https://github.com/FutureNorthants/northants-design-system/pull/114
- https://github.com/FutureNorthants/northants-website/pull/299

## Testing

- Checkout the design system branch, then run `yalc publish`
- Checkout the backend PR and import config
- Checkout this branch on the frontend and run `yalc add northants-design-system && yarn install && yarn dev`
- In a new window, run `yarn codegen` and `git status` to ensure there are no changes to the generated graphql.tsx file
- Visit the page you've previously edited in the CMS to include a google map slice and expect it to show up correctly
- Edit the page in the CMS to remove the slice, save/publish and refresh in the frontend, expect to see the change
- Verify home page, any page with hero image, any page with promotions slice, any page with video slice all still work as before too
